packages/webdriverio/src/constants.js

```diff
@@ -146,6 +153,20 @@ export const WDIO_DEFAULTS = {
             return true
         }
     },
+    /**
+     * set of WDIO services to use
+     */
+    services: {
+        type: 'string[]',
```

Comment:
this can also be an object when you provide an not published service, e.g.

```js
services: [new MyCustomService()]
```

Reply:
actually, see second example here: https://webdriver.io/docs/customservices.html